source/ssb-chart/templates/access-ingress-internal-authorization-policy.yaml

```diff
@@ -0,0 +1,96 @@
+{{- if and .Values.access (and .Values.access.ingress .Values.access.ingress.internal) }}
+{{- range .Values.access.ingress.internal }}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: "{{ template "app.name" $ }}-ingress-{{.namespace }}-{{ .application }}-auth-policy"
+  labels:
+{{ include "default.labels" $ | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "app.name" $ }}
+  action: ALLOW
+  rules:
+    {{- $fromPrincipal := printf "cluster.local/ns/%s/sa/%s-sa" .namespace .application }}
+  {{- if not .rules }}
+    - from:
+        - source:
+            principals: [ "{{ $fromPrincipal }}" ]
+  {{- else }}
+    {{- range .rules }}
+    - from:
+        - source:
+            principals: [ "{{ $fromPrincipal }}" ]
+            {{- if .jwt }}
+            {{- /* The request identity is in the format of "<ISS>/<SUB>" */}}
+            requestPrincipals: [ "{{ .jwt.issuer }}/*" ]
+            {{- end}}
+{{/*      If .paths or .methods are defined*/}}
+      {{- if or .paths .methods }}
+      to:
+        - operation:
+
+            {{- if .paths}}
+            paths:
+              {{- range .paths }}
+              - "{{ . }}"
+              {{- end }}
+            {{- end }}
+            {{- if .methods}}
+            methods:
+              {{- range .methods }}
+              - "{{ . }}"
+              {{- end }}
+            {{- end }}
+      {{- end }}
+{{/*     TODO: Consider if 'when' should be supported       */}}
```

Comment:
For internal ingress `when` might not be that useful, but for external ingress it is very useful.
We might end up using the same template for internal and external ingress, so it might be placed here.